en/docs/learn/adding-users-and-roles.md

```diff
@@ -1,10 +1,16 @@
 # Adding Users and Roles
 
-## Scenario
-This tutorial demonstrates user and role management in WSO2 Identity Server. User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
+User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
 
 This tutorial demonstrates how to create a new user, create a role with a set of permissions, and then log in to the dashboard as the newly created user. 
 
+## Scenario
+
+Pickup is a cab company that has many employees. Each of these employees have a particular designation such as "Manager", 
+"Driver" etc., and depending on their designation within the company, they will have different permissions and access levels. 
```

Comment:
The sentence is split into two lines.

Reply:
Fixed with aef9696f95c1cfc17cb8624c017f1f43833c82c8